internal/database/database.go

```diff
@@ -0,0 +1,49 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	_ "modernc.org/sqlite" // Import the sqlite3 driver
+)
+
+type Service struct {
+	DB *sql.DB
+}
+
+func Conect(DBFile string) (*Service, error) {
+	var install bool
+
+	if _, err := os.Stat(DBFile); os.IsNotExist(err) {
+		fmt.Println(DBFile)
+		install = true
+	}
+
+	// Open the database
+	db, err := sql.Open("sqlite", DBFile)
+	if err != nil {
+		return nil, err
```

Comment:
Лучше все ошибки оборачивать, чтобы быстрее находить где они случились
```suggestion
		return nil, fmt.Errorf("error while open db file %s: %w", DBFile, err)
```